VK_App/VK_App/ViewControllers/FirstBarVC/FriendsPhotosViewController.swift

```diff
@@ -70,7 +88,9 @@ final class FriendsPhotosViewController: UIViewController {
         networkService.fetchUserPhotos(for: userID) { [weak self] results in
             switch results {
             case let .success(photoPaths):
-                self?.photoNames = photoPaths.map(\.url)
+                let photoOptional = photoPaths.response.photos.map(\.photos.last)
+                self?.photoNames = photoOptional.map { $0?.url ?? "NO" }
```

Comment:
```suggestion
                self.photoNames = photoOptional.map { $0?.url ?? "NO" }
```
другие селфы также закинь в guard конструкции и удали "?"